src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        String[] inputFiles = {"apple.csv", "banana.csv", "grape.csv", "orange.csv"};
```

Comment:
use git rm --cached Main.java, it shouldn't be in PR

Reply:
fix comment 
https://github.com/mate-academy/jv-work-with-file/pull/1515/files#r1770556880

Reply:
done, but I'm not sure that it's done all right